spring-cloud-skipper/spring-cloud-skipper-server/src/test/java/org/springframework/cloud/skipper/server/db/migration/AbstractSkipperSmokeTest.java

```diff
@@ -78,14 +88,47 @@ public void testStart() {
 		assertThat(deployerData.getId()).isNotEqualTo(0);
 		assertThat(deployerData.getDeploymentDataAsMap()).isNotEmpty();
 		assertThat(deployerData.getDeploymentDataAsMap()).containsEntry("a", "b");
+
+		Release release = createRelease();
```

Comment:
Is this related to the PR or just increasing test coverage?

Reply:
Expanding test to cover #5715